docs/components/navigation/index.html

```diff
@@ -45,7 +45,7 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
               <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
             </hx-disclosure>
             <hx-reveal id="section-1-2" open>
-              <a href="#">Link 3-1</a>
+              <a href="#" class="active">Link 3-1</a>
```

Comment:
To show this state in our documentation, you can add documentation level styles.

Reply:
Changes have been done, Please review